packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -22,13 +22,21 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  url_launcher_platform_interface: ^1.0.4
+  url_launcher_platform_interface:
+    git:
+      url: https://github.com/balvinderz/plugins
+      ref: windowname
+      path: packages/url_launcher/url_launcher_platform_interface
```

Comment:
This makes the plugin unpublishable. Once we figure out all the rest of the changes, you'll have to split this PR in 3: one for each of the packages you've touched (and merge them in the correct order)